api/v2/endpoints/post-identity-map.md

```diff
@@ -123,6 +123,54 @@ A successful decrypted response returns the UID2s and salt bucket IDs for the sp
 }
 ```
 
+When some identifiers are considered invalid we will use "unmapped" list to deliver them. In this case, the response status remains "success".
+Note that "unmapped" list does not appear when all identifiers have been mapped.
```

Comment:
Suggest replace this line with the above which includes this info.

Reply:
great, changed!